java-tic-tac-toe.iml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I thought so too, but looked it up and found: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839. I think that moving to .idea project structure may be the recommended way forward.